universal/data/nyse_n.csv

```diff
@@ -1,4 +1,4 @@
-A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,Q,R,S,T,U,V,W
+A,B,C,D,E,F,G,H,I,J,K,L,M,N,O,P,x1,R,S,T,U,V,W
```

Comment:
Is there a reason for this change?

Reply:
There is `OLS.from_formula` fails if there is a `Q` column in a dataset. You're welcome if you have an idea how to solve it better :)

Reply:
Ohh right, `Q` is a reserved keyword in patsy... I like this hack, don't think there's an easy way around it.